server/portal/templates/base.html

```diff
@@ -16,8 +16,9 @@
     <link rel="canonical" href="{{request.get_full_path}}">
     <base href="/">
     {% block head_extra %}{% endblock %}
-    <!-- styles -->
+    <!-- CMS Assets -->
     <link id="css-site-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <script id="css-site-script" type="module" src="/static/site_cms/js/site.css.js"></script>
```

Comment:
If the CMS baked in the theme variable values, then this is **not** necessary, because Portal already loads CMS header CSS.

Reply:
Done. Moot.